### DISCUSSION TITLE
Kanban Board Update and Invite

### DISCUSSION THREAD

**Discussion by Dovineowuor** [Joymutheu-dev/MISSING-PERSONS-DATABASE-2024-KENYA-FINANCE-BILL-PROTESTS-#33] [General]
# Discussion Announcement: Collaboration Invitation for the Lost In Kenya Database Project

Hello Team,

I hope this message finds you well!

I’d like to invite everyone to participate in a collaborative discussion regarding our **Lost In Kenya Database** project. Your input and insights are invaluable as we work towards refining and enhancing the features and functionality of our application.

#### Key Discussion Points:
- **Feature Development**: Ideas for new features and improvements.
- **Bug Reports and Fixes**: Identifying and resolving any issues.
- **Integration and Usability**: Ensuring smooth integration with USSD and WhatsApp reporting, and improving user experience.
- **Project Roadmap**: Updates and planning for upcoming milestones.

Please visit the [Project Kanban Board](https://github.com/users/Dovineowuor/projects/14/views/1) to review current tasks and contribute your thoughts. Your active participation will help us drive this project forward and make a meaningful impact.

Feel free to add comments, suggest changes, and engage in discussions directly on the board. Let’s collaborate to make the Lost In Kenya Database as effective and user-friendly as possible!

Thank you for your continued support and contributions.

Best regards,

Dovie K

---

**Comment by eugengi** [comment]
Hey @Dovineowuor this is a good idea!
I support this to streamline work, onboard contributors, and better communicate to the community what the team is working on📢
I can create a simple [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects) template to get started. We can iterate on it over time

**Comment by eugengi** [comment]
It appears the link you've shared is probably for a _private_ repo and so isn't accessible🔏

**Reply by Dovineowuor** [reply]
Let me publish it
